.woodpecker/.verify-pr.yml

```diff
@@ -1,14 +1,14 @@
 pipeline:
   install:
-    image: danlynn/ember-cli:4.4.0
+    image: danlynn/ember-cli:4.12.1
```

Comment:
It's possible that this changed the node version (which might have issues with the lock file if it also brings in a newer npm). I would verify if the node in this image matches the one provided by madnificent/docker-ember (which we use in the Dockerfile). I think the madnificent one still uses node 16.

Reply:
The ember version itself is not really important and don't need to match. Global ember-cli will always use the local project version when available.

Reply:
> It's possible that this changed the node version (which might have issues with the lock file if it also brings in a newer npm). I would verify if the node in this image matches the one provided by madnificent/docker-ember (which we use in the Dockerfile). I think the madnificent one still uses node 16.

Just checked, it uses node 18, will update the image